api/src/hyp3_api/openapi-spec.yml

```diff
@@ -215,22 +216,19 @@ components:
 #       - INSAR_GAMMA
```

Comment:
might as well re-enable INSAR_GAMMA at this point, although that might mean some additional test cases, so could do it as a separate PR